typing/jkind.ml

```diff
@@ -1390,6 +1396,28 @@ module Const = struct
         name = "word mod everything"
       }
 
+    let bits8 =
+      { jkind = mk_jkind (Base Bits8) ~mode_crossing:true ~nullability:Non_null;
+        name = "bits8"
+      }
+
+    (* CR layouts v3: change to [Maybe_null] when separability is implemented. *)
+    let kind_of_unboxed_int8 =
+      { jkind = mk_jkind (Base Bits8) ~mode_crossing:true ~nullability:Non_null;
```

Comment:
`~nullability:Non_null` is set on unboxed types so we can still put them into arrays. Perhaps it makes sense to define `bits8`/`bits16` as `Maybe_null` and thus rule out their arrays on the type level?